dianna/visualization/image.py

```diff
@@ -10,13 +10,14 @@ def _determine_vmax(max_data_value):
     return vmax
 
 
-def plot_image(heatmap,
-               original_data=None,
-               heatmap_cmap=None,
-               heatmap_range=(None, None),  # (vmin, vmax)
-               data_cmap=None,
-               show_plot=True,
-               output_filename=None):
+def plot_image(
```

Comment:
why this reformatting? just some autoformatter I suppose? it makes the pr a bit larger.